### ISSUE TITLE
Use property instead of getter methods.

### ISSUE THREAD

**Issue by oika** [chiptiptech/connect#23]
In JobManagerInfo, members aren't hidden from others, and are directly accessed from JobManager. ( `get_xx` isn't used. )
Use property to provide values for outside of the class.

```python
    def __init__(self, mac_addr):
        self.__mac_addr = mac_addr

    @property
    def mac_addr(self):
        return self.__mac_addr
```

**Comment by rasshai** [comment]
Thanks. Yes, this should be fixed. Can you do it?

**Comment by oika** [comment]
Yes, I will.

**Comment by rasshai** [comment]
Are you fixing StreamingConf with setters and getters as well?

**Comment by oika** [comment]
No, I'm not for now. StreamingConf doesn't have public variables except `conf` , so I think we have only to hide conf as `__conf` .

**Comment by oika** [comment]
I found that both ClusterInfo and TaskManager build `TaskManagerInfo` separately with all parts of StreamingConf (address, mac, port, device_type, slots and data_interfaces).  
Why don't we make `get_tm_info` in StreamingConf?
